packages/e2e-tests/fixtures/blocks/core__pullquote__main-color.parsed.json

```diff
@@ -1,22 +1,29 @@
 [
-	{
-		"blockName": "core/pullquote",
-		"attrs": {
-			"customMainColor": "#2207d0",
-			"textColor": "subtle-background",
-			"className": "has-background is-style-default"
-		},
-		"innerBlocks": [],
-		"innerHTML": "\n<figure class=\"wp-block-pullquote has-background is-style-default\" style=\"border-color:#2207d0\"><blockquote class=\"has-text-color has-subtle-background-color\"><p>Pullquote custom color</p><cite>my citation</cite></blockquote></figure>\n",
-		"innerContent": [
-			"\n<figure class=\"wp-block-pullquote has-background is-style-default\" style=\"border-color:#2207d0\"><blockquote class=\"has-text-color has-subtle-background-color\"><p>Pullquote custom color</p><cite>my citation</cite></blockquote></figure>\n"
-		]
-	},
-	{
-		"blockName": null,
-		"attrs": {},
-		"innerBlocks": [],
-		"innerHTML": "\n",
-		"innerContent": [ "\n" ]
-	}
+    {
+        "blockName": "core/pullquote",
+        "attrs": {
+            "backgroundColor": "blue",
+            "textColor": "white",
+            "style": {
+                "border": {
+                    "color": "#2207d0"
+                }
+            },
+            "className": "is-style-default"
+        },
+        "innerBlocks": [],
+        "innerHTML": "\n<figure class=\"wp-block-pullquote is-style-default has-border-color has-white-color has-blue-background-color has-text-color has-background\" style=\"border-color:#2207d0\"><blockquote><p>Pullquote custom color</p><cite>before block supports</cite></blockquote></figure>\n",
+        "innerContent": [
+            "\n<figure class=\"wp-block-pullquote is-style-default has-border-color has-white-color has-blue-background-color has-text-color has-background\" style=\"border-color:#2207d0\"><blockquote><p>Pullquote custom color</p><cite>before block supports</cite></blockquote></figure>\n"
+        ]
+    },
+    {
+        "blockName": null,
+        "attrs": {},
+        "innerBlocks": [],
+        "innerHTML": "\n",
+        "innerContent": [
+            "\n"
+        ]
+    }
```

Comment:
Ah, another one of [those](https://github.com/WordPress/gutenberg/pull/30951/files?diff=split&w=1#r615777377) -- this time pre-existing 🤔

Reply:
Maybe the same as [above](https://github.com/WordPress/gutenberg/pull/30951#pullrequestreview-638772735). I'll test my hunch later.